ShopifySharp.Tests/ShippingZone_Tests.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ShopifySharp.Tests
+{
+    [Trait("Category", "ShippingZone")]
+    public class ShippingZone_Tests
+    {
+        private ShippingZoneService _Service => new ShippingZoneService(Utils.MyShopifyUrl, Utils.AccessToken);
+
+        [Fact]
+        public async Task Lists_ShippingZones()
+        {
+            var list = await _Service.ListAsync();
+
+            Assert.NotNull(list);
```

Comment:
A expanded the test a little bit. I don't know how deep should I go. Should I test also the list of countries, provinces, priceBasedShippingRates etc?